src/Persistence/Utils/FileUtils.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Utils;
+
+internal static class FileUtils
+{
+    /// <summary>
+    /// Converts backslashes to forward slashes, removes trailing slashes, and converts to lowercase.
+    /// </summary>
+    /// <param name="path"></param>
+    /// <returns></returns>
+    public static string NormalizePath(string path)
+    {
+        return path.Trim().Replace('\\', '/').Trim('/').ToLowerInvariant();
```

Comment:
This normalize pattern is specific to Zip entries

Reply:
updated the code, managed to remove the reference to this method from the tests project